src/main/java/frc/lib/util/swerve/SwerveModule.java

```diff
@@ -103,7 +103,7 @@ private void resetToAbsolute() {
     /**
      *
      *
-     * @param rotationSpeed Drive motor speed (-1 <= value <= 1)
+     * @param rotationSpeed Drive motor speed (-1 &lt;= value &lt;= 1)
```

Comment:
eww 🤣

Reply:
Note that in vscode, it automatically turns these into `<` for hovering. Only place this is ugly in the actual source code.